src/main/java/blackjack/domain/card/Card.java

```diff
@@ -0,0 +1,27 @@
+package blackjack.domain.card;
+
+public class Card {
+    public static final String BLANK = " ";
+    private final Suit suit;
+    private final Symbol symbol;
+
+    public Card(Suit suit, Symbol symbol) {
+        this.suit = suit;
+        this.symbol = symbol;
+    }
+
+    public int getScore() {
+        return symbol.getScore();
+    }
+
+    @Override
+    public String toString() {
+        return suit.getName() +
+                BLANK +
+                symbol.getName();
+    }
+
+    public boolean isAce() {
+        return this.symbol.name().equals("ACE");
```

Comment:
값을 꺼내서 비교하지 말고 symbol 객체에게 메시지를 던져서 물어보는게 좋겠네요
```suggestion
        return this.symbol.isAce();
```